src/views/bootablevolumes/actions/components/EditBootableVolumesModal.tsx

```diff
@@ -83,7 +83,9 @@ const EditBootableVolumesModal: FC<EditBootableVolumesModalProps> = ({
     const instanceLabel = instanceType && {
       [DEFAULT_INSTANCETYPE_LABEL]: `${instanceType}.${size}`,
     };
-    const descriptionAnnotation = description && { [ANNOTATIONS.description]: description };
+    const descriptionAnnotation = description
```

Comment:
i wonder if simply 
const descriptionAnnotation = { [ANNOTATIONS.description]: description };
will do the trick?

Reply:
No, because, when user deletes the description, it becomes an empty string `''`. And that's what we don't wanna save. We want not to see description annotation in the yaml in this case. And `undefined` does the trick for us in this case perfectly. Having `description: ''` set for the volume does not make sense. It's unnecessary.

Reply:
Ok, so, in this case, I recommend using our isEmpty function on the description or trim, as a user enters simply space it will result in a truthy value.

Reply:
or we are ok with simply empty space(s) as a description? i think we better off.

Reply:
> or we are ok with simply empty space(s) as a description? i think we better off.

I think we have some function to remove spaces at the beginning and end of the string, but it doesn't work well, especially if the user enters more spaces and nothing else. I've already tested. But I can add it here to `description` anyway, it's some improvement, for sure.

> Ok, so, in this case, I recommend using our isEmpty function on the description or trim, as a user enters simply space it will result in a truthy value.

I don't understand how this could help us. IMO it doesn't hep us because we still do need the condition I've added, for checking the description's falsey value, because we do need to update the description annotation even when the value is falsey - to be able to remove the annotation in case that there's nothing, no description (or empty string). I'd leave it as it is, as it does exactly what we want, and just would add the function to remove unnecessary spaces from the description string I've already mentioned.

Reply:
Updated the PR according to the suggestions, PTAL ;) Thanks!